paddle/fluid/framework/ngraph_operator.cc

```diff
@@ -42,6 +57,7 @@ typedef enum {                /* nGraph support state on ops          */
                PARTIAL_TEST   /* Support partial list of ops for test */
 } op_state;
 
+// perform graph build through bridge and execute computation
 class NgraphOperator {
```

Comment:
why is this not inheriting from OperatorBase?

Reply:
I think `FusedOperator` and `NgraphOperator` are very confused name.